openmls/src/treesync/mod.rs

```diff
@@ -426,14 +435,14 @@ impl TreeSync {
     pub(crate) fn from_ratchet_tree(
         backend: &impl OpenMlsCryptoProvider,
         ciphersuite: Ciphersuite,
-        ratchet_tree: RatchetTree,
+        ratchet_tree: &RatchetTree,
```

Comment:
Why not keeping a move here and that way you avoid the additional clone everytime `from_ratchet_tree` is used?

Reply:
Because anyway the tree was deep copied (node are cloned) so the move semantics were not required here. But I reverted